doc/README.open_ce_build.md

```diff
@@ -410,3 +416,37 @@ The Dockerfile for this runtime image is located in [`images/opence-runtime/Dock
 This file can also be built and run manually and supports GPUs if the system is set up with
 `nvidia-container-runtime`. When building the Dockerfile directly, it does require a few arguents,
 check the Dockerfile for details.
+
+### Building packages with Power10 MMA optimization
+
+#### System Requirement
+System: RHEL 8.5 or above
+OS: Linux
+Power Architecture: Power9/Power10
+GCC Compiler: GCC10 and GCC11
+
+One can build Power10 enabled packages on Power9 or Power10 with above system requirements. To install GCC 10 or 11, following command can be used -
```

Comment:
> One can build Power10 enabled packages on Power9 or Power10 with above system requirements.

I suggest to highlight this further:  
One can build Power10 enabled packages with above system requirements. Note that Power10  is _not_ required on your build system. The libraries can be built on Power9 as well.

Reply:
Sounds good!